### PR TITLE
test: add PR template

### PR DESCRIPTION
**Pull request by blairdrummond**
**CONTRIBUTIONS WELCOME, PUSH DIRECTLY TO THIS BRANCH**

Adds a checklist of tests to fill out on new PRs

https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+# Tests / Quality Checks
+
+## Automated Testing/build and deployment
+- [ ] Does the image pass CI successfully (build, pass vulnerability scan, and pass automated test suite)?
+- [ ] If new features are added (new image, new binary, etc), have new automated tests been added to cover these?
+
+## Official Languages
+
+- [ ] Test all components JupyterLab components in French
```
Comment (ca-scribner) [inline .github/pull_request_template.md:13]: Is it possible that we break this?  Or is this more a "if we add something, is it in French too" kind of check?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## JupyterLab extensions
+
+- [ ] Does git work?
```
Comment (ca-scribner) [inline .github/pull_request_template.md:17]: This might be automatable?  I'll ask Jess about it

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- [ ] Does VS Code open?
+
+## VS Code tests
+
+- [ ] Can you install extensions?
+
+## Port/proxy
+
+Copy paste this into a python notebook:
```
Comment (ca-scribner) [inline .github/pull_request_template.md:26]: Is this something that has broken before?  It just felt obscure.  It is an awkward one to test so we could add some instructions here (tell people to deploy the image, open in kubeflow, etc.)?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        self.send_response(HTTPStatus.OK)
+        self.end_headers()
+        self.wfile.write(b'Hello world')
+
+
+httpd = socketserver.TCPServer(('', 8000), Handler)
+httpd.serve_forever()
+```
+
+Open a tab with `https://kubeflow.covid.cloud.statcan.ca/notebook/$NAMESPACE/$SERVER/proxy/8000`
```
Comment (Ito-Matsuda) [inline .github/pull_request_template.md:45]: So they would have the deploy tag on as well?

**Review by ca-scribner — APPROVED** [review]

**Pull request merged**